src/Recorder/OpenTracingRecorder.php

```diff
@@ -26,10 +26,13 @@ public function init(array $params): void
         }
         $this->tracer = $tracer;
 
+        // fallback to empty array if getallheaders() isn't available
+        $headers = function_exists('getallheaders') ? getallheaders() : [];
```

Comment:
Even with the `getallheaders` polyfill loaded by Composer, this function is undefined when the Laravel application bootstraps itself after a Lambda "re-boot".

I've even verified that the required file is loaded (via `get_included_files()`), but the function is still undefined. So gracefully falling back to an empty array seemed to be the safest course of action.